src/providers/sh/commands/remove.js

```diff
@@ -177,13 +177,13 @@ async function remove({ token, sh: { currentTeam } }) {
   })
 
   if (matches.length === 0) {
-    console.error(error(
-      `Could not find ${argv.safe
+    console.log(
+      `> ${chalk.yellow('Warning!')} Could not find ${argv.safe
         ? 'unaliased'
         : 'any'} deployments matching ${ids
         .map(id => chalk.bold(`"${id}"`))
         .join(', ')}. Run ${chalk.dim(`\`now ls\``)} to list.`
-    ))
+    )
     return process.exit(1)
```

Comment:
I would have thought that it's the `exit(1)` that makes the build fail - not whether it prints to `stderr/stfout`

Reply:
Oh, you are right. Fixed. Thanks!